### ISSUE TITLE
Single Word Recognition

### ISSUE THREAD

**Issue by kneeonsteve** [react-native-voice/voice#195]
# Question
Has anyone run into an issue with recognizing single words?  So, if I say "Yes", or "Food"...I'm getting exception failures, or in the rare case that it doesn't fail, the recognition is really off.  I'll say the word "For" and it come back to me with "IV"....I'd understand something like "Four, or Fore"...but roman numerals?  "Three" regularly returns "Wine".  Very confusing.

This might just be my build, but in case others have seen this I wanted to through it out there.  Sorry if repeat/bad question.  Thanks for any info!

**Comment by Ostefanini** [comment]
I have also run into this issue, on Android specifically (not tested yet on iOS)

I wonder if it's the last Google application update with the offline speech recognition that broke this ?

I have also notice that during some recognition session, that last part of the sentence isn't triggered via onSpeechPartialResult.

In addition, it seems that other event's recognition, like onSpeechRecognized aren't triggered with data.

**Comment by kneeonsteve** [comment]
Yes!  I have see the same "dropping the last word" instance happening on Android as well.  

In my testing (Android on Samsung S9, iOS on iPhone8+):
Android:  Has consistent issues with processing single words, especially numbers.  Will also regularly drop the last word in an utterance.  When it does work, accuracy is fairly high.

iOS:  Will pick up single words, but accuracy is really bad on them.  Doesn't seem to have the issue where the last word is dropped.  Multi-word utterance accuracy is not as good as Android.

Glad I'm not the only one seeing this!  May have to migrate to a cloud service for the voice piece.

**Comment by lfoliveir4** [comment]
@kneeonsteve Did you solved your problem?

**Comment by kneeonsteve** [comment]
@Ifoliveir4, Due to time, we had to go another route and use google’s speech recognition service...

**Comment by lfoliveir4** [comment]
@kneeonsteve Got it, I'll make some updates to the library and let's fix your issue.

**Comment by AdrianDiazCode** [comment]
This issue is really a thing, it's a great library but most single word commands aren't recognized at all, not even throwing bad alternatives

**Comment by anghelmarius** [comment]
I have same issue

**Comment by phongbksneep** [comment]
still issue

**Comment by phongbksneep** [comment]
issue still exists
"@react-native-voice/voice": "3.2.0"
    "react": "16.13.1",
    "react-native": "0.63.4",

**Comment by cutenesss** [comment]
anyone solved this?

**Comment by safaiyeh** [comment]
This project has been deprecated and archived. Please migrate to [`expo-speech-recognition`](https://github.com/jamsch/expo-speech-recognition). Thank you for your contributions! ❤️
